flink-core/src/main/java/org/apache/flink/api/common/state/ListState.java

```diff
@@ -27,10 +27,14 @@
  * The state is accessed and modified by user functions, and checkpointed consistently
  * by the system as part of the distributed snapshots.
  *
- * <p>The state is only accessible by functions applied on a {@code KeyedStream}. The key is
- * automatically supplied by the system, so the function always sees the value mapped to the
- * key of the current element. That way, the system can handle stream and state partitioning
- * consistently together.
+ * <p>The state can be a keyed list state or an operator list state.
+ * When it is a keyed list state, it is accessed by functions applied on a {@code KeyedStream}.
```

Comment:
we can add a `<p>` before `When it is a keyed list state` and `When it is an operator list state` to get better format?

Reply:
good suggestion!